src/views/Apply.vue

```diff
@@ -12,7 +14,8 @@
         />
       </div>
       <div class="form-item">
-        <p>Year:</p>
+        Year
+        <br />
 
         <input type="radio" id="form-year-1" v-model="year" value="1" />
```

Comment:
Can we add more spacing between the buttons here?

Reply:
sure